e2e/wdio/core/tests/switch.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import {
+    click,
+    elementDisplayed,
+    getAttributeByName,
+    getElementArrayLength,
+    getElementClass,
+    refreshPage,
+    waitForPresent
+} from '../../driver/wdio';
+import { switchPo } from '../../core/pages/switch.po';
+
+describe('Switch test suite', () => {
+
+    const switchPage = new switchPo();
+    const {
+        switchSizes, switchSizesExample, toggle,
+        toggleInput, switchBindingExample, switchFormsExample, semanticswitchExample,
+        playGroundSwitchExample, disabledToggle, acceptIcon, declineIcon, semanticSwitch,
+        switchBtn, checkboxes
+    } = switchPage;
+
+    beforeAll(() => {
+        switchPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(switchSizes);
+    }, 1);
+
+    it('Should check turn on/ Turn off switch toggle', () => {
+        checkSwitchingWork(switchSizesExample);
+        checkSwitchingWork(switchFormsExample);
+        checkSwitchingWork(semanticswitchExample);
+        checkSwitchingWork(switchBindingExample);
+        checkSwitchingWork(playGroundSwitchExample);
+    });
+
+    it('Should check icons on semantic toggles', () => {
+        for (let i = 0; i < 2; i++) {
+            expect(elementDisplayed(declineIcon, i)).toBe(true);
```

Comment:
add fail message as discussed on another PR
expect(elementDisplayed(declineIcon, i)).toBe(true, 'decline icon not displayed');